docs/docs/develop/writing-jobs/word-count.md

```diff
@@ -129,6 +111,13 @@ $ ../../gradlew execute
 
 ```
 
+### Old Implementation
+If you find the new operator API limiting, you can continue using existing RxJava based implementation. It's available in [WordCountJob.java](https://github.com/Netflix/mantis/blob/master/mantis-examples/mantis-examples-wordcount/src/main/java/com/netflix/mantis/examples/wordcount/WordCountJob.java).
+A few callouts using this approach are 
+1. keyBy is not distributed and is local to the worker
```

Comment:
We should have more documentation for things like these. Are they explained in more details in Java docs? If so, link to them. We need to have pretty detailed Java doc on each of these methods similar to RxJava Observable Java doc.